### DISCUSSION TITLE
Can I customise the enzyme settings for a Trypsin/LysC mix?

### DISCUSSION THREAD

**Discussion by ShaanSubramaniam** [vdemichev/DiaNN#759] [Q&A]
Hello,

I am using a Trypsin/LysC mix for sample preparation.

I realised that LysC can cleave K-P residues (Prolines c terminal of Lysine) but not R-P residues (Prolines c terminal of Arginine).

Is there a way to customise the specificity of the enzymes to reflect this during in silico library generation?

Many thanks!

**Comment by LeeSCantrell** [comment]
In command line options, try
```
--cut R*,!*P"
--cut K*
```
Note that Trypsin technically may still cut K-P.
